environment.yml

```diff
@@ -0,0 +1,21 @@
+name: myenv
```

Comment:
see name comment above

Reply:
Just changing this in github won't cause any merge conflicts, right?

Reply:
I've built the environment, and I'll change it here, but I'm gonna need to put it in the .gitignore, until I clone, delete, prune, and update.